frontend/packages/core/src/utils/generateBreadcrumbsEntries.tsx

```diff
@@ -1,23 +1,39 @@
-import type { Location } from "react-router-dom";
+import { Location, matchPath } from "react-router-dom";
 
+import type { Workflow } from "../AppProvider/workflow";
 import type { BreadcrumbEntry } from "../Breadcrumbs";
 
-const generateBreadcrumbsEntries = (location: Location, validateUrl: (url: string) => boolean) => {
+const HOME_ENTRY = { label: "Home", url: "/" };
+
+const generateBreadcrumbsEntries = (workflowsInPath: Array<Workflow>, location: Location) => {
```

Comment:
Can you add some comments going through the logic here with some examples and input/output.